.buildkite/lint

```diff
@@ -32,6 +32,10 @@ set -eu
 # Enable debug output, only enable temporarily, rather verbose
 #set -x
 
+# We need to update the master to determine the merge base
+# For now, trust the post-checkout hook to have done this.
+# git fetch origin master
```

Comment:
So why not just fetch again to make sure we have the latest master? Is there a drawback?